docs/input-jdbc.asciidoc

```diff
@@ -282,9 +282,13 @@ JDBC connection string
 ===== `jdbc_default_timezone`
 
   * Value type is <<string,string>>
+  ** Value should be a canonical timezone or offset, such as `Europe/Paris` or `Etc/GMT+3`
+  ** Value _may_ include square-bracketed extensions, such as `America/Denver[dst_enabled_on_overlap:true]`
   * There is no default value for this setting.
 
-Timezone conversion.
+[id="plugins-{type}s-{plugin}-jdbc_timezone_conv"]
+Timezone conversion
+
 Logstash (and Elasticsearch) expects that timestamps are expressed in UTC terms.
```

Comment:
```suggestion
Logstash and Elasticsearch expect timestamps to be expressed in UTC terms.
```

Reply:
I know that this sentence predated this PR and I understand why it was written this way. Still seems like a good opportunity to make an update. I don't think anybody who is not using ES will be confused by this statement, and it gives us an opportunity to clear up some grammar ambiguity.